gui/src/components/onboarding/pages/body-proportions/BodyProportions.tsx

```diff
@@ -128,28 +128,32 @@ function ProportionItem({
             'duration-300'
           )}
         >
-          <Tooltip
-            content={
-              <Localized id={`${part.label}-desc`}>
-                <Typography
-                  variant="standard"
-                  whitespace="whitespace-pre-wrap"
-                ></Typography>
-              </Localized>
-            }
-            preferedDirection="bottom"
-            mode="corner"
-          >
-            <div className="flex flex-grow" onClick={toggleOpen}>
-              <Typography variant="section-title" bold>
-                {l10n.getString(part.label)}
-              </Typography>
-            </div>
-          </Tooltip>
+          <div className="flex flex-grow" onClick={toggleOpen}>
+            <Typography variant="section-title" bold>
+              {l10n.getString(part.label)}
+            </Typography>
+            <Tooltip
+              content={
+                <Localized id={`${part.label}-desc`}>
+                  <Typography
+                    variant="standard"
+                    whitespace="whitespace-pre-wrap"
+                  ></Typography>
+                </Localized>
+              }
+              preferedDirection="bottom"
+              mode="corner"
+            >
+              <div className="info-icon hover:opacity-100 opacity-65 ml-1 scale-[0.65] border-2 border-solid text-xs w-5 h-5 flex justify-center items-center rounded-full">
```

Comment:
```suggestion
              <div className="hover:opacity-100 opacity-65 ml-1 scale-[0.65] border-2 border-solid text-xs w-5 h-5 flex justify-center items-center rounded-full">
```
that class doesnt exist

Reply:
![image](https://github.com/user-attachments/assets/963beea4-3acc-4b64-a2de-d57ced49355a)